src/backend/tests/unit/test_views.py

```
@@ -382,19 +383,19 @@ def test_update_template_manual_time(self, mock_auth, job_templates):
 
     def test_update_manual_costs(self, mock_auth):
         client = APIClient()
-        data = dict(type="manual", value=1000)
+        data = dict(monthly_subscription_cost=5000, engineer_avg_hourly_rate=1000)
         response = client.post(f"/api/v1/costs/", content_type='application/json', data=json.dumps(data))
```

Ruff (0.15.2)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)